app/src/main/res/values/strings.xml

```diff
@@ -594,4 +594,8 @@
 	<string name="button_key_f11">F11</string>
 	<!-- Text for the "F12" button in virtual keyboard. -->
 	<string name="button_key_f12">F12</string>
+	<string name="connection_type">Connection type:</string>
```

Comment:
You need to add a comment above all these to give translators context for each string. They can't see the UI most of the time, so it has to describe what it's for and where it's used.